### ISSUE TITLE
Stats Issue Dashboard is Inaccurate

### ISSUE THREAD

**Issue by sdzhong** [getsentry/sentry#76025]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

I would like to better understand what the Stats > Issues > `All Unresolved Issues` should be calculating. The other graph `New and Returning Issues` doesn't appear accurate either. 
**Can someone please confirm the equations/calculations used for each of the graphs in Stats > Issues?**

### Expected Result

Matching data/results between Stats > Issues and Issues stream search.

### Actual Result

![Image](https://github.com/user-attachments/assets/65656de6-9f1f-4e80-864f-981e505ae3a0)

### Product Area

Stats

**Comment by brianthi** [comment]
Hi @sdzhong, the stats page shows usage data for all events ingested by sentry for your entire organization. The issues stream shows issues which have aggregated events that roll up into distinct issues. In the simplest use case, if you set up Sentry as new customer and send 2 error events that both aggregate into the same issue, you would see one issue on the issue stream, with event count of 2 and on the stats page 2 errors sent for that project. Here's [a link to our docs](https://docs.sentry.io/product/stats/) with more details about how the stats page works and what data it shows.

**Comment by sdzhong** [comment]
@brianthi 
> the stats page shows usage data for all events ingested by sentry for your entire organization. The issues stream shows issues which have aggregated events that roll up into distinct issues.

I presumed the Stats > Issues page shows data in the context of Issues, which is what the issues stream does as well. Is that correct? If so, should this GitHub Issue be reopened?

**Comment by brianthi** [comment]
Ah my apologies, I didn't notice that your question was about the issues tab within the stats page. I believe you'll need to ensure the team filter is on the issues stream to have a closer match. I'll forward this issue to the team that manages this issues tab for them to confirm whether or not if all the filters match exactly, that those numbers should be correct. The copy in the stats page is slightly confusing as it references "in the last 7 days" when there's a date filter that allows going beyond 7 days.

**Comment by sdzhong** [comment]
> I believe you'll need to ensure the team filter is on the issues stream to have a closer match. 

Yep, this should have been done as described in the original post, along with the other criteria

**Comment by rachrwang** [comment]
Will work with @mrduncan to get a response back to you - thanks!

**Comment by ReneGreen27** [comment]
Another example and related to internal ticket [#154919](https://sentry.zendesk.com/agent/tickets/154919)

**Comment by gaprl** [comment]
Hey @sdzhong, this issue has been open for over a year. Is this still relevant? If we don't hear back in the next couple of weeks, we'll close this issue.

If this is still happening, could you provide:
1. Current screenshots showing the discrepancy
2. The specific filters/parameters you're using in the Issues stream for comparison
3. The expected vs actual numbers you're seeing

**Comment by gaprl** [comment]
Hey @sdzhong, Gabriel from Sentry here following up on this issue. It's been over a year since you reported this discrepancy in the Stats dashboard.

I've reviewed the discussion and it seems there's still confusion about how the Stats > Issues page calculations work compared to the Issues stream. To move this forward, I need to understand if this is still affecting you.

Could you please:
1. Confirm if you're still experiencing these discrepancies
2. If yes, provide updated screenshots showing the current differences you're seeing
3. Share the specific filters/parameters you're using in both the Stats page and Issues stream for comparison
4. Provide the expected vs actual numbers you're observing

If we don't hear back in the next couple of weeks, we'll close this issue as stale. Thanks for your patience!

**Comment by guemi** [comment]
@gaprl 

Not the initial poster, but can confirm I am seeing the same thing.

Last night at 23.00 I got 19 errors in an application, according to the Sentry UI:

<img width="267" height="194" alt="Image" src="https://github.com/user-attachments/assets/2f820521-b7d4-4537-91f4-d1180d537f73" />


But querying the Stats API with the Sentry Grafan Plugin shows only 12:

<img width="1376" height="285" alt="Image" src="https://github.com/user-attachments/assets/9d8275b7-61b4-494e-8703-78711464a951" />

<img width="162" height="119" alt="Image" src="https://github.com/user-attachments/assets/9e3b828c-4f6d-4763-aa12-191132554799" />


Surely that can't be intended?

**Comment by sdzhong** [comment]
Hey @gaprl , 

> Could you please: 
> Confirm if you're still experiencing these discrepancies

Yes

> If yes, provide updated screenshots showing the current differences you're seeing

<img width="1716" height="1284" alt="Image" src="https://github.com/user-attachments/assets/e676f1cd-5cbc-4c1c-bebf-b5b4b2eafb6a" />

[Stats & Usage > Issues](https://team-se.sentry.io/settings/stats/issues/?pageStatsPeriod=2w&team=4511096495603712) vs [Issues Feed](https://team-se.sentry.io/issues/?project=-1&statsPeriod=14d) 

> Share the specific filters/parameters you're using in both the Stats page and Issues stream for comparison

Please see attached screenshot, red text

> Provide the expected vs actual numbers you're observing

Please see attached screenshot, green text
